requests/src/requests/Requester.scala

```diff
@@ -367,6 +358,15 @@ case class Requester(verb: String,
     }
   }
 
+  /**
+    * Do something with an OutputStream and close it
+    * @param os OutputStream
+    * @param fn 
+    */
+  private def withOs[T](os: OutputStream)(fn: OutputStream => T) : Unit = 
+    try fn(os) finally os.close()
+  
+
```

Comment:
The Scaladoc here doesn't add much value.
Also I suggest you to rename the method to `usingOutputStream` to be consistent with Scala 2.13's `Using`
```suggestion
  private def usingOutputStream[T](os: OutputStream)(fn: OutputStream => T) : Unit = 
    try fn(os) finally os.close()

```